canary/environments/pull-environments.sh

```diff
@@ -2,8 +2,9 @@
 set -e
 IFS='|'
 
-# In development, AWS_PROFILE should be set. In CI, it's not.
-[ "$AWS_PROFILE" ] && useProfile="true" || useProfile="false";
```

Comment:
Previously, CI didn't use AWS profiles. Now we're using profiles with CI too, due to known issue (#11009).